cou/utils/juju_utils.py

```diff
@@ -75,7 +80,8 @@ async def async_set_current_model_name(model_name: Optional[str] = None):
     return CURRENT_MODEL_NAME
 
 
-async def _async_get_model(model_name=None) -> Model:
+# pylint: disable=global-statement, global-variable-not-assigned
```

Comment:
We should removed line 95 `global CURRENT_MODEL_NAME` instead of using `global-variable-not-assigned`

Reply:
done